tests/application/users.test.js

```diff
@@ -1,5 +1,7 @@
 const axios = require('axios');
 const nock = require('nock');
+require('dotenv').config();
```

Comment:
This isn't necessary for integration tests, the credentials are entirely fake, as are the HTTP requests used in these tests.